trello_api.py

```diff
@@ -0,0 +1,48 @@
+import requests
+from config import api_key, token, board_id, todo_list_id, doing_list_id, done_list_id
+
+
+class TrelloApi:
+
+    def __init__(self):
+        self.api_key = api_key
+        self.token = token
+        self.board_id = board_id
+        self.todo_list_id = todo_list_id
+        self.doing_list_id = doing_list_id
+        self.done_list_id = done_list_id
+
+    def get_board_cards(self):
+        endpoint = f"https://api.trello.com/1/boards/{self.board_id}/cards?"
+        response = requests.get(endpoint, params={"key": self.api_key, "token": self.token}).json()
+        return response
+
+    def get_board_lists(self):
+        endpoint = f"https://api.trello.com/1/boards/{board_id}/lists?"
+        response = requests.get(endpoint, params={"key": self.api_key, "token": self.token}).json()               
+        return response
+
+    def get_list_cards(self, card_id):
+        endpoint = f"https://api.trello.com/1/cards/{card_id}/list?"
+        response = requests.get(endpoint, params={"key": self.api_key, "token": self.token}).json()
+        return response
+
+    def add_card(self, new_todo_item):
+        endpoint = f"https://api.trello.com/1/cards?name={new_todo_item}"
+        response = requests.post(endpoint, params={"key": self.api_key, "token": self.token, "idList": self.todo_list_id}).json()

```

Comment:
It's confusing to include one query parameter via your `endpoint` string and then include the other 3 via the `params` dictionary.
The end result is the same wherever you put the query parameter, but splitting it up is harder to read and could lead to problems.
A dictionary is slightly easier to read/manage than building a long string, so I'd suggest moving `name` there.